packages/core/lib/core.ts

```diff
@@ -84,6 +88,9 @@ export function createClient<S extends CoreSchema, C extends Configuration> (opt
     startSpan: (name, spanOptions?: SpanOptions) => {
       const span = spanFactory.startSpan(name, spanOptions)
       return spanFactory.toPublicApi(span)
+    },
+    get currentSpanContext () {
+      return spanContextStorage.current
```

Comment:
It'd be good to add some tests for this. Should be possible by passing a storage as an `option` and pushing to that

Reply:
yeah good point - I was going to test this 'for real' that after implementing `makeCurrentContext`, but it's good to have a test that uses the storage instance provided in client options as well